integrations/argocd/.port/resources/port-app-config.yaml

```diff
@@ -69,7 +69,7 @@ resources:
     selector:
       query: "true"
     port:
-      itemsToParse: .status.history
+      itemsToParse: .status.history // []
```

Comment:
strange comment here ?

Reply:
<img width="1000" alt="Screenshot 2024-04-26 at 1 24 26 PM" src="https://github.com/port-labs/ocean/assets/15999660/655a3fca-6953-488d-b5b7-de52cbe69fe0">

Add the `// []` takes care of these warnings that spams the log